spec/rubocop/cop/style/raise_args_spec.rb

```diff
@@ -142,6 +142,11 @@
       expect(cop.offenses).to be_empty
     end
 
+    it 'accepts exception constructor with keyword arguments' do
+      inspect_source(cop, 'raise RuntimeError.new(a: 1, b: 2)')
```

Comment:
Good point. I'll change the others that give multiple arguments to `RuntimeError` while I'm correcting this.